components/Carousel/Carousel.js

```diff
@@ -1,8 +1,41 @@
 class Carousel {
+    constructor(carousel) {
+        this.carousel = carousel;
+        this.imgs = this.carousel.querySelectorAll("img");
+        this.leftBtn = this.carousel.querySelector(".left-button");
+        this.rightBtn = this.carousel.querySelector(".right-button");
+        this.index = 0;
+        this.imgs[this.index].style.display = "block";
+
+        this.rightBtn.addEventListener("click", () => this.toggleRight());
+        this.leftBtn.addEventListener("click", () => this.toggleLeft());
+    }
+    toggleRight() {
+        this.imgs[this.index].style.display = "none";
+
+        if (this.index ===  this.imgs.length-1) {
+            this.index = 0;
+        } else {
+            this.index++
+        }
+        this.imgs[this.index].style.display = "block";
+    }
+
+    toggleLeft() {
+        this.imgs[this.index].style.display = "none";
+
+        if (this.index === 0) {
+            this.index = this.imgs.length -1;
+        } else {
+            this.index--;
+        }
+        this.imgs[this.index].style.display = "block";
+    }
```

Comment:
https://github.com/brilles/Sprint-Challenge-Applied-Javascript/blob/6775cb1e176752f09d35fedecae4f534690e2d24/components/Carousel/Carousel.js#L13-L32

Excellent implementation of the logic here.